app/src/ts.app.js

```diff
@@ -55,7 +58,7 @@ class Listener {
 				return this;
 			}
 			let content = JSON.parse(e.data.replace(BROADCAST_PREFIX, ''));
-			if (content.key !== key) {
+			if (content.key !== key || content.appIds !== this.appIds) {
```

Comment:
so this is where we should try to use the glob syntax later on, right now we should just have support for `*` or a specific appId